app/src/main/java/at/team30/setroute/Helper/NavigationHelper.kt

```diff
@@ -0,0 +1,45 @@
+package at.team30.setroute.Helper
+
+import at.team30.setroute.models.Route
+import com.google.android.gms.maps.model.LatLng
+
+class NavigationHelper {
+    companion object {
+        val linkBase : String = "https://www.google.com/maps/dir/?api=1&"
+
+        fun getNavLink(route : Route): String {
+            var link : String = linkBase
+            var originString = ""
+            var destinationString = ""
+            var waypointString = ""
+
+            val travelMode = "travelmode=walking"
```

Comment:
can be moved to a static member as well.

right below `val linkBase: String ....`:

```
val travelMode = "travelmode=walking"
```